src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/AzureTarget.tsx

```diff
@@ -0,0 +1,312 @@
+            onClick={() => {
+              setManual(false);
+              clearState();
+            }}
+          />
+          <Radio
+            name="azure-share-manual"
+            id="azure-share-manual"
+            label="Manually enter the account information."
+            checked={manual}
+            isChecked={manual}
+            onClick={() => {
+              setManual(true);
+              clearState();
+            }}
+          />
+        </FormGroup>
+        {!manual && (
+          <>
+            <SourcesSelect
+              provider="azure"
+              selectedSourceId={sourceId}
+              setSelectedSourceId={setSourceId}
+              setFetchingError={setIsErrorFetchingSources}
+            />
+            <SourcesButton />
+            <>
+              {isErrorFetchingSources && (
+                <Alert
+                  variant={'danger'}
+                  isPlain={true}
+                  isInline={true}
+                  title={'Sources unavailable'}
+                >
+                  Sources cannot be reached, try again later or enter an account
+                  info for upload manually.
+                </Alert>
+              )}
+              {(isErrorFetchingSources || isErrorFetchingDetails) && (
+                <Alert
+                  variant={'danger'}
+                  isPlain
+                  isInline
+                  title={'AWS details unavailable'}
+                >
+                  Could not fetch Tenant id and Subscription id from Azure for
+                  given Source. Check Sources page for the source availability
+                  or select a different Source.
+                </Alert>
+              )}
+            </>
+          </>
+        )}
+        {!manual && (
+          <>
+            <Gallery hasGutter>
+              <GalleryItem>
+                <FormGroup label="Azure Tenant GUID" isRequired>
+                  <TextInput
+                    value={tenantId}
+                    type="text"
+                    readOnlyVariant="default"
+                    aria-label="Azure Tenant GUID"
+                    validated={
+                      tenantId
+                        ? ValidatedOptions.default
+                        : ValidatedOptions.error
+                    }
+                    isRequired
+                  />
+                </FormGroup>
+              </GalleryItem>
+              <GalleryItem>
+                <FormGroup label="Subscription ID" isRequired>
+                  <TextInput
+                    value={subscriptionId}
+                    type="text"
+                    readOnlyVariant="default"
+                    aria-label="Subscription ID"
+                    isRequired
+                  />
+                </FormGroup>
+              </GalleryItem>
+            </Gallery>
+            <AzureAuthButton tenantId={tenantId} />
+            <FormGroup label="Resource group" isRequired>
```

Comment:
Does not show "No results found when there are no results. 

original: 
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/3fa25e69-d2b1-4b1d-8b25-b367cbc4560d)

current:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/287b030e-95d2-48d8-9211-c192038e961b)


It could also be disabled when there is no source selected I think?

Reply:
I think the no results found may be a better option, I need to learn how to make that happen.